draft-ietf-core-resource-directory.md

```diff
@@ -679,7 +679,7 @@ Location: /rd/4521
 A Resource Directory may optionally support HTTP. Here is an example of the same registration operation above, when done using HTTP.
 
 ~~~~
-Req: POST /rd?ep=node1 HTTP/1.1
+Req: POST /rd?ep=node1&con=http://[2001:db8::1:1] HTTP/1.1
```

Comment:
I chose the default port for the example's server (and then elided it); the ephemeral port would be the one the client used for sending the POST request, and thus would be the default value that needs to be explicitly set or overridden in con=. (I'm not sure whether a TCP stack would allow accepting connections on the client port; if yes, the client could explicitly send it in con, but IMO that'd be too unusual a setup to show in an example.)